cdap-ui/app/cdap/components/ExperimentalBanner/ExperimentalBanner.scss

```diff
@@ -36,6 +36,8 @@
     top: 100%;
     border-width: 5px 5px;
     border-style: solid;
-    border-color: $brand-primary-color-lighter $brand-primary-color-lighter transparent transparent;
+    // TODO: Need to figure out how to use CSS
```

Comment:
Negative. We should not do any `lighten` or `darken`. We should specify specific color